mlrun/model_monitoring/applications/base.py

```diff
@@ -127,53 +211,78 @@ def _handler(
         start: Optional[str] = None,
         end: Optional[str] = None,
         base_period: Optional[int] = None,
+        write_output: bool = False,
+        stream_profile: Optional[ds_profile.DatastoreProfile] = None,
```

Comment:
with what value of stream_profile the _handler is called in run_local = false?

Reply:
In a remote run, `stream_profile` is simply `None`. The profile is fetched from the env vars + k8s secrets